src/common/options/rbd.yaml.in

```
@@ -481,6 +481,10 @@ options:
   type: uint
   level: advanced
   desc: default order (data block object size) for new images
+  long_desc: This configures the default object size for new rbd images. The value is used as
+    power to two, meaning ``default_object_size = 2 ^ rbd_default_order`` (alternative
+    format ``default_object_size = 1 << rbd_default_order``). Configure a value between 12 and 25
```

The `1 << ` syntax is fairly niche, most people are confused by it.  Let's drop it.
```suggestion
    Configure a value between 12 and 25
```